versions/release/1.16.3/config.json

```diff
@@ -0,0 +1,26 @@
+{
+    "mcinjector": {
+        "version": "de.oceanlabs.mcp:mcinjector:3.8.0:fatjar",
+        "args": ["--in", "{input}", "--out", "{output}", "--log", "{log}", "--level=INFO", "--lvt=LVT", "--exc", "{exceptions}", "--acc", "{access}", "--ctr", "{constructors}"]
+    },
+    "fernflower": {
+        "version": "net.minecraftforge:forgeflower:1.5.478.15",
```

Comment:
I was hopeing we could bump this to `1.5.478.16` for 1.16.3

Reply:
There is no reason to.